curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646e44246c224501ee59542f.md

```diff
@@ -0,0 +1,210 @@
+---
+id: 646e44246c224501ee59542f
+title: Step 18
+challengeType: 0
+dashedName: step-18
+---
+
+# --description--
+
+In JavaScript, the `split()` method is used to split up a string into an array of substrings. This method also comes with an optional separator parameter which is a type of pattern that tells the computer where each split should happen.
+
+ "coca-cola".split("-") // this would return [ 'coca', 'cola' ]
```

Comment:
this needs to be in a code block like so

````
```js
"coca-cola".split("-") // this would return [ 'coca', 'cola' ]
```
````